### DISCUSSION TITLE
To disable file open on by key 'l' or <ENTER>

### DISCUSSION THREAD

**Discussion by pritam12426** [jarun/nnn#1885] [Q&A] [answered]
In [nnn v2.2](https://github.com/jarun/nnn/releases/tag/v2.2) @jarun  introduce an ENV `DISABLE_FILE_OPEN_ON_NAV` then -> in version [nnn v2.3](https://github.com/jarun/nnn/releases/tag/v2.3) it change to ENV `NNN_RESTRICT_NAV_OPEN`. Now my question is, Is there is any new way to get this feature in version `v4.9`

**Comment by N-R-K** [comment]
> Is there is any new way to get this feature in version `v4.9`

There isn't. But you can edit keybindings in `src/nnn.h` and compile it.

**Comment by pritam12426** [comment]
@N-R-K Could you provide some examples? It would make it easier to understand.

**Reply by N-R-K** [reply, marked as answer]
Just remove this line from `src/nnn.h` and recompile:

```c
	{ 'l',            SEL_NAV_IN },
```

It will disable <kbd>l</kbd>.
